.drone.star

```diff
@@ -0,0 +1,1324 @@
+config = {
+	'app': 'updater',
+	'rocketchat': {
+		'channel': 'builds',
+		'from_secret': 'private_rocketchat'
+	},
+
+	'branches': [
+		'master'
+	],
+	'yarnlint': True,
```

Comment:
```suggestion
	'phplint': True,
```

yarn is a JavaScript thing. So `phplint` will be a better name. Rename wherever it is used.

Reply:
fixed